### DISCUSSION TITLE
Where do I get IONOS api key

### DISCUSSION THREAD

**Discussion by helgeerbe** [helgeerbe/certbot-dns-ionos#4] [Q&A] [answered]
Hi,

How do you get API information from IONOS ? dns_ionos_prefix and dns_ionos_secrets

Regards

**Comment by helgeerbe** [comment, marked as answer]
It's actual an api beta featue and you have to sign up for it. But this is quite easy. Follow the instructions: https://developer.hosting.ionos.de/docs/getstarted

**Reply by clementmas** [reply]
I think this link should be added to the readme. That's the first thing I was looking for.

Thanks for this package!
